src/Training/Bundle/UserNamingBundle/Integration/Form/Type/TransportSettingsFormType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Training\Bundle\UserNamingBundle\Integration\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Training\Bundle\UserNamingBundle\Entity\FakeTransportSettings;
+
+class TransportSettingsFormType extends AbstractType
+{
+    const NAME = 'training_fake_transport_setting_form_type';
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add(
+            'url',
+            UrlType::class,
+            [
+                'label' => 'oro.website.url.label',
```

Comment:
It's a bad practice to reuse translation messages as it limits translators from defining different messages for different places.